pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1013,14 +1013,14 @@ func hardwareHealthTimerTask(ctx *zedagentContext, handleChannel chan interface{
 	// Run a timer for extra safety to send hardwarehealth updates
 	// If we failed with the initial we have a short timer, otherwise
 	// the configurable one.
-	const shortTime = 120 // Short time: two minutes
+	const shortTimeSecs = 120 // Short time: two minutes
 	hardwareHealthInterval := ctx.globalConfig.GlobalValueInt(types.HardwareHealthInterval)
-	interval := time.Duration(hardwareHealthInterval) * time.Second
+	interval := time.Duration(hardwareHealthInterval)
```

Comment:
I think it is not a good idea to use time.Duration without the proper time unit.
Once variable is of type time.Duration, it should not need any further multiplication by time unit.
Integer intervalSecs or what was here before would make more sense.